Backendless/Service/UserService.cs

```diff
@@ -354,6 +354,39 @@ public void ResendEmailConfirmation( String email, AsyncCallback<Object> callbac
       }
     }
 
+#if !( NET_35 || NET_40 )
+    public async Task<Object> CreateEmailConfirmationAsync( String identity )
+    {
+      return await Task.Run( () => CreateEmailConfirmation( identity ) ).ConfigureAwait( false );
+    }
+#endif
+
+    public Object CreateEmailConfirmation( String identity )

```

Comment:
Is this the method that returns URLs for the email confirmation link?

Reply:
Yes